src/legacy/core_plugins/data/public/search/expressions/esaggs.ts

```diff
@@ -288,6 +288,11 @@ export const esaggs = (): ExpressionFunction<typeof name, Context, Arguments, Re
         const cleanedColumn: KibanaDatatableColumn = {
           id: column.id,
           name: column.name,
+          _meta: {
+            type: column.aggConfig.type,
+            indexPattern: column.aggConfig.getIndexPattern(),
```

Comment:
Not sure whether that's out of scope for this PR, but the index pattern object is not serializable, right? It's also pretty big: https://github.com/elastic/kibana/pull/55351/files#diff-c85d790e1f9fa1b5d3c3ca92ea62bd60R1

Should this be just the id of the pattern?

Reply:
I was almost going to add a comment suggesting you name this `indexPatternId` to avoid confusion (I thought it _was_ the id).